README.md

```diff
@@ -179,8 +179,8 @@ A pre-built Docker image is available at https://hub.docker.com/r/wisvch/florrie
 To build your own Docker image, first create a tarball of the repository including all npm dependencies:
```

Comment:
These instructions are outdated, the Dockerfile doesn’t need pre-built dependencies.

Reply:
Done